migration/src/main/scala/akka/persistence/r2dbc/migration/MigrationTool.scala

```diff
@@ -123,6 +123,12 @@ class MigrationTool(system: ActorSystem[_]) {
   private val targetPluginId = migrationConfig.getString("target.persistence-plugin-id")
   private val targetR2dbcSettings = R2dbcSettings(system.settings.config.getConfig(targetPluginId))
 
+  targetR2dbcSettings.dialectName
+  require(
+    !targetR2dbcSettings.durableStateAssertSingleWriter,
+    "When running the MigrationTool the " +
```

Comment:
"When" vs "While"? (Once migration is done it can be set to on, right?)

Reply:
👍